### PR TITLE
Document BrowserView.{destroy,isDestroyed}

### LINKED ISSUE #10096: No way to destroy a BrowserView
Right now when you're done with a `BrowserView` you can manually call `view.webContents.destroy()` and null out all your references, but checking the number of processes I'm still seeing an extra one.

### PR DESCRIPTION
**Pull request by poiru**
This fixes #10096.

Note that we haven't addressed [this comment](https://github.com/electron/electron/issues/10096#issuecomment-320163081) yet. I'll do that in a separate PR.

### CODE REVIEW

File: docs/api/browser-view.md [hunk 1]
```diff
@@ -73,6 +73,16 @@ A `Integer` representing the unique ID of the view.
 
 Objects created with `new BrowserView` have the following instance methods:
 
+#### `view.destroy()`
+
+Force closing the view, the `unload` and `beforeunload` event won't be emitted
```
Comment (YurySolovyov) [inline docs/api/browser-view.md]: event*s*

Comment (poiru) [reply docs/api/browser-view.md]: Fixed!

File: spec/api-browser-view-spec.js [hunk 2]
```diff
@@ -30,6 +30,22 @@ describe('BrowserView module', () => {
...
 
+  describe('BrowserView.destroy()', () => {
+    it('does not throw', () => {
+      view = new BrowserView()
+      view.destroy()
+    })
+  })
+
+  describe('BrowserView.isDestroyed()', () => {
+    it('does not throw', () => {
```
Comment (MarshallOfSound) [inline spec/api-browser-view-spec.js]: does not throw?

This should be `returns correct destoryed status` or something

Comment (poiru) [reply spec/api-browser-view-spec.js]: @MarshallOfSound Fixed!

**Pull request merged**